docs/_source/tutorials_and_integrations/tutorials/feedback/end2end_examples/add-suggestions-and-responses-005.ipynb

```diff
@@ -0,0 +1,519 @@
+    "\n",
+    "![ui-responses](../../../../_static/tutorials/end2end/text-classification/argilla-annotation-ui-responses.png)\n",
+    "\n",
+    "As all the records are submitted, we no longer see the suggestion for each one of the records. To demonstrate how they are seen, you can see the Argilla UI below where the record is not submitted yet and the suggestion (`Sports` in this case) is shown to the annotator.\n",
+    "\n",
+    "![ui-suggestions](../../../../_static/tutorials/end2end/text-classification/argilla-annotation-ui-suggestions.png)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### To HuggingFace Hub\n",
+    "\n",
+    "If you would like to push the dataset you created to the HuggingFace Hub, you can simply use the `push_to_huggingface` method to upload it. Do not forget to create a model card as well, which will make the dataset more readable and understandable for the users.\n",
+    "\n",
+    "To be able to upload your dataset to the Hub, you must be logged in to the Hub. The following cell will log us with our previous token.\n",
+    "\n",
+    "If we don't have one already, we can obtain it from [here](https://huggingface.co/docs/hub/security-tokens) (remember to set the write access)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from huggingface_hub import login\n",
+    "\n",
+    "login(token=hf_token)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We only need to call the `push_to_huggingface` method to push the dataset to HuggingFace Hub. If we have a dataset with the same name on the Hub, this method will update the existing one."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataset.push_to_huggingface(name=\"end2end_textclassification_with_suggestions_and_responses\", generate_card=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Conclusion\n",
+    "\n",
+    "In this tutorial, we have seen how we can add responses and suggestions to our dataset. Adding responses to your dataset given that you already have the annotated labels is a great way to save time and effort for your project. Similarly, adding suggestions to your dataset will make the annotation process much faster and easier for your annotators. We have seen how we can employ these tools for both `FeedbackDataset` and `RemoteFeedbackDataset`.\n",
+    "\n",
+    "***ADD LINK FOR NEXT TUTORIAL***"
```

Comment:
I think we should update this section. We can now typically train a model, compute metrics. Feel free to already create an empty notebook for this.

Also, feel free to add a reference to a filtering and sorting notebook for the metadata tutorial.